<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMuestraImg</class>
 <widget class="QDialog" name="frmMuestraImg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>484</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Imagen del linde</string>
  </property>
  <property name="toolTip">
   <string>Muestra la imagens del linde</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="lbImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Imagen del linde</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bttAnterior">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>450</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Anterior imagen seleccionada</string>
   </property>
   <property name="text">
    <string>Anterior imagen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttSiguiente">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>450</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Siguiente imagem seleccionada</string>
   </property>
   <property name="text">
    <string>Siguiente imagen</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblDatos">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>261</width>
     <height>401</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Datos del punto</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <column/>
  </widget>
  <widget class="QLabel" name="lbMensaje">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>831</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagen del linde</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Datos del punto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbEstado">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
